x-pack/plugin/ilm/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/ilm/actions/DownsampleActionIT.java

```diff
@@ -109,16 +109,24 @@ public void refreshAbstractions() {
         );
     }
 
-    private void createIndex(String index, String alias) throws IOException {
+    @Before
+    public void updateClusterSettings() throws IOException {
+        updateClusterSettings(client(), Settings.builder().put("indices.lifecycle.poll_interval", "5s").build());
```

Comment:
Is this something we want? What's the purpose of delaying the poll interval to later than the 1 second we configure it in tests?

Reply:
For some reason the `@After cleanUpCluster()` method sets the following: 
```
[2023-03-29T13:19:01,928][INFO ][o.e.c.s.ClusterSettings  ] [runTask-0] updating [indices.lifecycle.poll_interval] from [5s] to [10m]
``` 

So, I need to reset it at my `@Before` test

Reply:
I just ran the test on your branch (after having removed this @Before method) and I don't see the setting being changed to `10m`.

Gradle configures the poll interval to `1000ms` for these test clusters https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/ilm/qa/multi-node/build.gradle#L29 (`elasticsearch.yml` for all nodes in the cluster will contain this setting)

This is my diff for your branch 
```
--- a/x-pack/plugin/ilm/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/ilm/actions/DownsampleActionIT.java
+++ b/x-pack/plugin/ilm/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/ilm/actions/DownsampleActionIT.java
@@ -109,11 +109,6 @@ public class DownsampleActionIT extends ESRestTestCase {
         );
     }

-    @Before
-    public void updateClusterSettings() throws IOException {
-        updateClusterSettings(client(), Settings.builder().put("indices.lifecycle.poll_interval", "5s").build());
-    }
-
     private void createIndex(String index, String alias, boolean isTimeSeries) throws IOException {
         Settings.Builder settings = Settings.builder()
             .put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, 1)
```

Could you remove this and try again please? I'm fairly convinced we don't need it